lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -271,14 +271,14 @@ function _campaign_resource_retrieve($nid) {
 /**
  * Retrieve and show index list response of campaigns requested.
  *
- * @param string $ids
- * @param bool $staff_pick
- * @param bool $mobile_app
- * @param string $mobile_app_date
- * @param string $term_ids
- * @param int $count
- * @param bool $random
- * @param int $page
+ * @param  string  $ids
```

Comment:
Yeah, not sure. I'm trying it. It's something I pulled from how Laravel does docblocks (which goes to extra lengths to make sure the code and comments look appealing and are easier to digest). Kinda curious if others find it useful/appealing.